packages/create-cloudflare/e2e-tests/pages.test.ts

```diff
@@ -148,193 +278,35 @@ describe.concurrent(`E2E: Web frameworks`, () => {
-					input: ["n"],
-				},
-			],
-			testCommitMessage: true,
-			timeout: 1000 * 60 * 6,
-		},
-		hono: {
-			expectResponseToContain: "Hello Hono!",
-			testCommitMessage: false,
-		},
-		qwik: {
-			expectResponseToContain: "Welcome to Qwik",
-			promptHandlers: [
-				{
-					matcher: /Yes looks good, finish update/,
-					input: [keys.enter],
-				},
-			],
-			testCommitMessage: true,
-		},
-		remix: {
-			expectResponseToContain: "Welcome to Remix",
-			testCommitMessage: true,
-			quarantine: true,
-		},
-		next: {
-			expectResponseToContain: "Create Next App",
-			promptHandlers: [
-				{
-					matcher: /Do you want to use the next-on-pages eslint-plugin\?/,
-					input: ["y"],
-				},
-			],
-			testCommitMessage: true,
-			quarantine: true,
-		},
-		nuxt: {
-			expectResponseToContain: "Welcome to Nuxt!",
-			overrides: {
-				packageScripts: {
-					build: "NITRO_PRESET=cloudflare-pages nuxt build",
-				},
-			},
-			testCommitMessage: true,
-		},
-		react: {
-			expectResponseToContain: "React App",
-			testCommitMessage: true,
-		},
-		solid: {
-			expectResponseToContain: "Hello world",
-			promptHandlers: [
-				{
-					matcher: /Which template do you want to use/,
-					input: [keys.enter],
-				},
-				{
-					matcher: /Server Side Rendering/,
-					input: [keys.enter],
-				},
-				{
-					matcher: /Use TypeScript/,
-					input: [keys.enter],
-				},
-			],
-			testCommitMessage: true,
-		},
-		svelte: {
-			expectResponseToContain: "SvelteKit app",
-			promptHandlers: [
-				{
-					matcher: /Which Svelte app template/,
-					input: [keys.enter],
-				},
-				{
-					matcher: /Add type checking with TypeScript/,
-					input: [keys.down, keys.enter],
-				},
-				{
-					matcher: /Select additional options/,
-					input: [keys.enter],
-				},
-			],
-			testCommitMessage: true,
-		},
-		vue: {
-			expectResponseToContain: "Vite App",
-			testCommitMessage: true,
-		},
-	};
-
 	Object.keys(frameworkTests).forEach((framework) => {
-		const config = frameworkTests[framework];
-		if (isQuarantineMode() || config.quarantine) return;
+		const { quarantine, timeout, testCommitMessage } =
+			frameworkTests[framework];
 
-		const skip = frameworkToTest && frameworkToTest !== framework;
+		const quarantineSkip = isQuarantineMode() !== (quarantine ?? false);
+		const singleFrameworkSkip =
+			frameworkToTest && frameworkToTest !== framework;
 
-		test.skipIf(skip)(
+		test.skipIf(quarantineSkip || singleFrameworkSkip)(
 			framework,
 			async () => {
-				await runCliWithDeploy(
-					framework,
-					frameworkTests[framework].testCommitMessage
-				);
-			},
-			{ retry: 3, timeout: config.timeout || TEST_TIMEOUT }
-		);
-	});
-
-	Object.keys(frameworkTests).forEach((framework) => {
-		const config = frameworkTests[framework];
-		if (!isQuarantineMode() || !config.quarantine) return;
-
-		const skip = frameworkToTest && frameworkToTest !== framework;
-
-		test.skipIf(skip)(
-			`Quarantined: ${framework}`,
```

Comment:
Since you are removing this test naming, I think we can/should remove this `replace` logic as well: https://github.com/cloudflare/workers-sdk/blob/b2fddfaaf83cd430987589170f54c462ea654b9f/packages/create-cloudflare/e2e-tests/pages.test.ts#L149
no?

___

Also you can also remove the whole `getFrameworkNameFromTestContext` thing altogether and revert back to the plain `ctx.meta.name` if you think it's not necessary, or keep it as a safety net, whatever you prefer 🙂